File: python/bee1018.py
while True:
    valor = int(input())
    print(valor)
    if 0 < valor < 1000000:
        notas = [100, 50, 20, 10, 5, 2, 1]
        for nota in notas:
            qnt_nota = int(valor / nota)
            print(f'{qnt_nota} nota(s) de R$ {nota},00')
            valor -= qnt_nota * nota
        break

File: java/bee1018.java
import java.util.Scanner;

public class bee1018 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int valor = input.nextInt();
        int notas[] = {100, 50, 20, 10, 5, 2, 1};

        System.out.println(valor);

        for(int i = 0; i < notas.length; i++) {
            int qntNotas = valor / notas[i];
            System.out.printf("%d nota(s) de R$ %d,00\n", qntNotas, notas[i]);
            valor %= notas[i];
        }

        input.close();
    }
}
